src/praisonai-train/praisonai_train/data/__init__.py

```
@@ -6,8 +6,10 @@
 """
 from praisonai_train.data import checks as _checks  # noqa: F401  registers checks
 from praisonai_train.data import recipes as _recipes  # noqa: F401  registers recipes
+from praisonai_train.data.benchmark import BenchResult, benchmark_deployments
 from praisonai_train.data.generate import generate_dataset
 from praisonai_train.data.qc import filter_rows, score
 from praisonai_train.data.registry import checks, recipes
 
-__all__ = ["generate_dataset", "score", "filter_rows", "recipes", "checks"]
+__all__ = ["generate_dataset", "score", "filter_rows", "recipes", "checks",
+           "benchmark_deployments", "BenchResult"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)